.gitignore

```diff
@@ -0,0 +1,4 @@
+test.md
```

Comment:
Kindly add a description for the PR with what you did exactly in that project in order for the PR to be professional 
you can edit this by pressing the three dots icon then press edit.
![image](https://user-images.githubusercontent.com/24703171/159301585-859b7fd9-da0a-4960-85c8-fb9c57884b0e.png)